src/liboslexec/llvm_ops.cpp

```diff
@@ -753,14 +753,14 @@ osl_normalize_vv (void *result, void *a)
     using std::sqrt;
```

Comment:
The `using std::sqrt` line can be removed then?

Reply:
Well what do you know? Somehow we both missed this, I guess.
I'm tempted to abandon this. Alex, do you have a reason why we should keep these changes?